README.md

```diff
@@ -141,7 +142,7 @@ Options that should be send as `initializationOptions` as part of `initialize` r
 
 ### Settings
 
-* `FSharp.keywordsAutocomplete` - provides keywords in autocomplete list, recommended default value: `true`
+* `FSharp.KeywordsAutocomplete` - provides keywords in autocomplete list, recommended default value: `true`
```

Comment:
the casing isn't quite right here - while FSAC parses settings with PascalCase, the settings are defined in the [Ionide-vscode package.json](https://github.com/ionide/ionide-vscode-fsharp/blob/main/release/package.json#L445) with lowercase first letters, and so that's what we should document here.

Reply:
ooh.. I thought it's the same with the FSharp dto definition. Let me revert these.